FRAuth/FRAuth/FRAuth.swift

```diff
@@ -86,6 +97,13 @@ public final class FRAuth: NSObject {
     /// - Throws: ConfigError
     static func initPrivate(config: [String: Any]) throws {
         
+        // Check if there is an existing session and destroy it, do a clean up as this would only happen at this point
+        // if the SDK has been previously initialized and a succesfull authentication has happened. Given that the SDK gets
+        // re-initiallised, possibly with a different configuration we need to do some clean up.
+        if let _ = FRUser.currentUser {
+            FRAuth.cleanUp()
+        }
+        
         // Validate server config
         guard let server = config["forgerock_url"] as? String,
```

Comment:
Inside this method we use a lot of hard coded config property names. Now that we have FROptions.CodingKeys enum, I think we should use that instead: **FROptions.CodingKeys.url** in this case.